src/common/components/game/FullscreenMessage.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { number, string } from 'prop-types';
+
+const FullscreenMessage = ({ message, height, background }) =>
+  <div style={{
+    position: 'absolute',
+    left: 0,
+    width: '100%',
+    height: height,
+    zIndex: 9999,
```

Comment:
It'd be nice if we had some sort of reasoning behind choosing these `z-index` numbers. I know it's `9999` to be on top of some other thing but it would be nice to have more concrete guidelines.

Reply:
Yeahhh.

I agree, but I don't know what a good solution would be. I guess maybe some constants?